tests/fixtures/starknet.py

```diff
@@ -230,11 +230,8 @@ def _factory(entrypoint, **kwargs) -> list:
             data = profile_from_tracer_data(tracer_data)
 
             output_path = (
-                str(
-                    request.node.path.parent
-                    / f"{request.node.path.stem}.{entrypoint}({(json.dumps(kwargs) if kwargs else '')})"
-                )[:220]
-                + ".pb.gz"
+                str(request.node.path.parent / f"{request.node.path.stem}.{entrypoint}")
```

Comment:
In case of a parametrized test, you want to have the args in the name to be able to identity the test case. Just shorten the 220 to something lore reasonable and that fits in your case ?

Reply:
Changed, removed the '(' as linux bash doesn't work with it.